plugins/palo_alto_pan_os/help.md

```diff
@@ -185,6 +193,10 @@ Example output:
 
 This action checks to see if an IP address, CIDR IP address, or domain is in an Address Group.
 
+##### Support
```

Comment:
Same as above

Reply:
@blaxminarayan-r7 You can probably just call this out once or twice and say "same with the others in this file" to save a bunch of typing :) The team will understand